.github/workflows/main.yml

```diff
@@ -144,6 +144,23 @@ jobs:
           signoff: true
           body: Add v${{ github.event.inputs.tag }} to community operators
           committer: quay-devel <quay-devel@redhat.com>
+  
+  pull-request-k8s-operatorhub:
```

Comment:
Don't we have to rerun the `Download manifests` step like in the `pull-request-redhat-openshift-ecosystem` step? I think the two steps should look identical except for the checkout and PR steps